content/_admin/ip-whitelisting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Egress IP Addresses & IP Whitelisting
+layout: article
+section: elastic.io Specific Settings
+description: "Document lists public IP addresses of our clusters to whitelist if required."
+order: 1
```

Comment:
@jhorbulyk we need this document 3rd in the list. Please change order to `3`.

Reply:
I'd put this new section 'elastic.io Specific Settings' into 'Integrator guide'. There open a new subsection 'Platform Behavior'. The existing article 'Platform Behavior' will be its first entry and renamed to 'General' or 'Overview'. And the second entry will be 'elastic.io Specific Settings'. I'd suggest to rename it to 'IP Addresses' though.

Reply:
I would disagree with @joscha-kuerten-mvise since the whole documentation is white-listed. Our OEM customers don't need this.

Reply:
> we need this document 3rd in the list. Please change order to 3.

What would change? The ordering looks the same in both cases:
![Screenshot from 2022-05-10 11-26-17](https://user-images.githubusercontent.com/5710732/167596490-c83062a3-0f31-4988-bcd1-6c2586528789.png)
![Screenshot from 2022-05-10 11-25-18](https://user-images.githubusercontent.com/5710732/167596493-6ac2c626-2324-4ae5-a094-af57b6dd6f2a.png)

Reply:
> The whole documentation is white-listed. Our OEM customers don't need this.

But app.elastic.io direct customers do need this. There have been some discussions about adding some sort of automated tagging to build documentation in different configurations. That might be helpful for this case.